package.json

```diff
@@ -62,6 +63,7 @@
     "lodash.omit": "^4.5.0",
     "query-string": "^6.13.6",
     "react": "^16.14.0",
+    "react-alice-carousel": "^2.0.2",
```

Comment:
not fully happy about this but damn, carousel components out there are bad and this had the least amount of issues. What I actually wanted is on large screens a 4x2 grid on each carousel pane, but 2x1 on small screens, and more in-between. This was not easily possible with this library either, so until we have 100 data partner assets it should suffice.